### PR TITLE
Added OrbitMc

### PR DESCRIPTION
**Pull request by RedolentHalo**
OrbitMc

### General:
* [ ] The pull request title is descriptive. *(ex. `Added OrbitMc`, not `Updated metadata.json`)*
* [ ] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [ ] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [ ] No changes were made to the file's formatting (4 spaces per tab).
* [ ] There are no syntax errors.
* [ ] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [ ] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: "orbitmc" 
  - [ ] `name`: "OrbitMc" 
  - [ ] `description`: "Place for ASIAN Pvpers" 
  - [ ] `addresses`: "mc.orbitmc.xyz"
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [ ] `primaryAddress`: "mc.orbitmc.xyz"
  - [ ] `minecraftVersions`:  ["1.7.*", "1.8.*"]
  - [ ] `primaryMinecraftVersion`: "1.8.9" 
  - [ ] `primaryColor`: "#FFD700" 
  - [ ] `secondaryColor`: "#FFFF00"
  - [ ] `primaryRegion`: "AS"
  - [ ] `regions`: ["AS"]
  - [ ] `gameTypes`: ["PVP", "MINIGAMES", "BEDWARS"]
  - [ ] (optional) `website`:
  - [ ] (optional) `store`: "https://store.orbitmc.xyz"

### Socials
* [ ] (optional) `twitter`: username of twitter account without the @ *(ex. MyServer)*
* [ ] (optional) `discord`: invite link to discord *(ex. https://discord.com/invite/4ceEJuH)*
* [ ] (optional) `youtube`: slug or username of youtube channel *(ex. MyServer)*
* [ ] (optional) `instagram`: username of instagram account *(ex. MyServer)*
* [ ] (optional) `twitch`: username of twitch account *(ex. MyServer)*
* [ ] (optional) `telegram`: slug of telegram group *(ex. MyServer)*
* [ ] (optional) `reddit`: slug of subdreddit wtih 'r/' *(ex. MyServer)*
* [ ] (optional) `tiktok`: username of tiktok account *(ex. MyServer)*
* [ ] (optional) `facebook`: slug of facebook page *(ex. MyServer)*

#### Logo
* [ ] My logo is a `png` file.
* [ ] I have uploaded my logo to my server folder orbitmc and named it `logo.png`.
* [ ] My logo has a transparent background and is square (1:1 aspect ratio).
* [ ] My logo is `4096` pixels in width and height.
* [ ] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [ ] My background is a `png` file.
* [ ] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [ ] My background has an aspect ratio of 16:9.
* [ ] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [ ] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [ ] My background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

File: servers/orbitmc/metadata.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "AS"   
+    ],
+    "gameTypes": [
+        "PvP",
+        "Bedwars"
+    ],
+    "website": "https://soon.soon.soon",
+    "store": "https://store.orbitmc.xyz",
+    "socials": {
+        "twitter": "None",
```
Comment (wi) [inline servers/orbitmc/metadata.json:33]: if there is no twitter/telegram don't specify none just remove it from the JSON.

**Review by wi — CHANGES_REQUESTED** [review]
Your background.png does not meet the minimum in the README.MD please review it.

File: servers/orbitmc/metadata.json (hunk 1 above)
Comment (RedolentHalo) [reply servers/orbitmc/metadata.json:33]: okok

**Comment by RedolentHalo** [comment]
alr done i guess

File: servers/orbitmc/metadata.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "regions": [
+        "AS"   
+    ],
+    "gameTypes": [
+        "PvP",
+        "Bedwars"
+    ],
+    "store": "https://store.orbitmc.xyz",
+    "socials": {
+        "discord": "https://discord.orbitmc.xyz"
```
Comment (wi) [inline servers/orbitmc/metadata.json:32]: This needs to be a discord.gg link

File: servers/orbitmc/metadata.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "id": "orbitmc",
+    "name": "OrbitMc",
+    "description": "Place for ASIAN Pvpers",
+    "addresses": [
+        "mc.orbitmc.xyz"
```
Comment (wi) [inline servers/orbitmc/metadata.json:6]: you don't need the subdomain in the addresses a rray

File: servers/orbitmc/metadata.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "name": "OrbitMc",
+    "description": "Place for ASIAN Pvpers",
+    "addresses": [
+        "mc.orbitmc.xyz"
+
+    ],
+    "primaryAddress": "mc.orbitmc.xyz",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*",
```
Comment (wi) [inline servers/orbitmc/metadata.json:12]: this isn't valid JSON

**Review by wi — CHANGES_REQUESTED** [review]

**Comment by RedolentHalo** [comment]
done

File: servers/orbitmc/metadata.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "AS",
+    "regions": [
+        "AS"   
+    ],
+    "gameTypes": [
+        "PvP",
+        "Bedwars"
```
Comment (wi) [inline servers/orbitmc/metadata.json:28]: Bedwars is not valid please change it with one of the following: 

['PvP', 'PvE', 'HCF', 'Factions', 'Minigames', 'Skyblock', 'Parkour', 'UHC', 'Hardcore', 'Survival', 'Open World', 'Prison', 'Creative', 'Roleplay', 'Adventure']

**Review by wi — CHANGES_REQUESTED** [review]

File: servers/orbitmc/metadata.json (hunk 5 above)
Comment (RedolentHalo) [reply servers/orbitmc/metadata.json]: is that it? i did it now please merge

**Review by wi — APPROVED** [review]

**Pull request merged**